scripts/safe_template_linter.py

```diff
@@ -399,6 +549,8 @@ def _check_mako_file_is_safe(self, mako_template, results):
             results: A file results objects to which violations will be added.
 
         """
+        if self._is_django_template(mako_template):
+            return
```

Comment:
(Limited template knowledge question) Is it possible for a django template to include a mako template that would need to be checked for safety? If so, `_check_include_files` should come before this `return`.